src/components/nodes/GLBViewerNode.tsx

```
@@ -0,0 +1,549 @@
+    }
+
+    return () => {
+      cancelled = true;
+      // Cleanup previous scene
+      if (sceneRef.current) {
+        sceneRef.current.traverse((obj) => {
+          if (obj instanceof THREE.Mesh) {
+            try { obj.geometry?.dispose(); } catch (e) { console.warn("GLB geometry dispose failed:", e); }
+            try {
+              if (Array.isArray(obj.material)) {
+                obj.material.forEach((m) => m.dispose());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)